docs/source/use_with_pytorch.mdx

```diff
@@ -0,0 +1,194 @@
+# Use with PyTorch
+
+## Tensors
+
+### Dataset format
+
+By default, datasets return regular python objects: integers, floats, strings, lists, etc.
+
+To get PyTorch tensors instead, you can set the format of the dataset to "pytorch" using [`Dataset.with_format`]:
```

Comment:
Small discrepancy with the `with_format` docstring which says it accepts `torch` instead of `pytorch`. Should we update the docstring to align this?

Reply:
Both are supported, but let me change this so that we use the same everywhere for consistency